### PR TITLE
Fix API documentation and site content 

### PR DESCRIPTION
**Pull request by suhothayan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Siddhi Execution Regex
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/siddhi-io/siddhi-execution-regex.svg)](https://github.com/siddhi-io/siddhi-execution-regex/commits/master)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-The **siddhi-execution-regex extension** is a <a target="_blank" href="https://siddhi.io/">Siddhi</a> extension that provides basic regex handling capabilities such as concat, length, replace all, etc.
+The **siddhi-execution-regex extension** is a <a target="_blank" href="https://siddhi.io/">Siddhi</a> extension that provides Provides basic RegEx execution capabilities such as find, match, etc.
```
Comment (ramindu90) [inline README.md]: @suhothayan, "provides" word has duplicated

Comment (pcnfernando) [inline README.md]: Seems 'provides' is duplicated

File: docs/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Siddhi Execution Regex
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/siddhi-io/siddhi-execution-regex.svg)](https://github.com/siddhi-io/siddhi-execution-regex/commits/master)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-The **siddhi-execution-regex extension** is a <a target="_blank" href="https://siddhi.io/">Siddhi</a> extension that provides basic regex handling capabilities such as concat, length, replace all, etc.
+The **siddhi-execution-regex extension** is a <a target="_blank" href="https://siddhi.io/">Siddhi</a> extension that provides Provides basic RegEx execution capabilities such as find, match, etc.
```
Comment (pcnfernando) [inline docs/index.md]: Duplicated 'Provides'

Comment (suhothayan) [reply docs/index.md]: fixed

File: README.md (hunk 1 above)
Comment (suhothayan) [reply README.md]: fixed

Comment (suhothayan) [reply README.md]: fixed

**Pull request merged**